src/EditorFeatures/CSharpTest/RefactoringHelpers/RefactoringHelpersTests.cs

```diff
@@ -1,10 +1,12 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System.Threading.Tasks;
+using ICSharpCode.Decompiler.CSharp.Syntax;
```

Comment:
very curious about this...

Reply:
Probably a mistake, it seems to me that when using refactorings to move things around certain things get added almost randomly. Will check